src/components/BrandConfigProvider/Styles.ts

```diff
@@ -0,0 +1,18 @@
+import { TextStyles } from '../Text/TextStyles';
+import { Theme } from './Theme';
+
+interface Props {
+  theme?: Theme;
+}
+
+export class Styles {
+  Text: TextStyles;
+
+  private _theme: Theme;
+
+  constructor({ theme }: Props = {}) {
+    this._theme = theme || new Theme();
+
+    this.Text = new TextStyles(this._theme);
```

Comment:
Right on - so all components' styles will need to be worked into this class.  And customers/we can subclass this within apps if we want to have more props (e.g. for components that are only in that app).  Let's be crazy careful about what goes in here - just want to avoid refactors that amount to breaking changes.  If we do that well, I think this is going to be a nice approach.

Reply:
Yes, I think that is this class's one job, to provide access to all the component styles, and thus will need a handle on them.  This could be inverted by having the component style objects register themselves with this class.  That would make it so no changes are needed here when a new component is added.